### DISCUSSION TITLE
Duvidas com page-dinamic-detail

### DISCUSSION THREAD

**Discussion by Msanfellice** [po-ui/po-angular#1863] [Tire suas dúvidas] [answered]
boa tarde pessoal, alguem poderia me dar um help, estou comecando a estudar o po-ui, e seguindo a documentacao do site, porem para quem esta iniciando o site e um pouco confuso, como por exemplo estou comecando com o componente Page Dynamic Detail, ja importei todos os componentes conforme a documentacao do site, mas mesmo assim ocorre uns erros que para quem esta comecando fica dificil de resolver
como por exemplo esse erro que estou tendo no meu projeto:
![erro_po](https://github.com/po-ui/po-angular/assets/133433500/aaea1dba-1fb3-4fa8-b261-84986c39f743)

**Comment by anliben** [comment, marked as answer]
Boa Tarde @Msanfellice, tudo bem?

Basta apenas colocar o seu componente no app.module.ts no objeto declarations e ele reconhecera o po-ui

Porem o PoModule deve esta no objeto imports

Segue guia de instalação: https://po-ui.io/guides/getting-started

Attr.

**Comment by anliben** [comment]
Qualquer coisa tenho um video no YouTube como começar PO-UI!

https://youtu.be/gS89PdrLk4A?si=oeOTxinRna0PPQ1X

**Comment by Msanfellice** [comment]
Obrigado

Em qua, 1 de nov de 2023 14:03, João Victor ***@***.***>
escreveu:
